src/2_RTB/VerbaliEsterni/VerbaleEsterno_241123/content.typ

```diff
@@ -0,0 +1,70 @@
+Il team ha presentato in modo pratico e conciso i risultati ottenuti: è stato attivato un file *Docker Compose*, all'interno del quale lo script di simulazione realizzato in Python, ed eseguito singolarmente in un container, inviava dati simulati a un'istanza di Kafka, anch'essa un container. 
+
+La proponete ha affermato che il prodotto mostrato corrispondeva pienamente alle loro aspettative per questo sprint.
+
+=== Progressi Documentali
+L'attenzione si è focalizzata sul documento di analisi dei requisiti.
+Si sono discussi i casi d'uso riportati all'interno del documento e il team ne ha approfittato per esprimere alcuni dubbi irrisolti. 
+
+Si riportano gli argomenti salienti: 
+- *Necessità di interagire con i Sensori*: il team ha espresso il il dubbio relativo all'inserimento di un caso d'uso che permettesse all'attore di interagire direttamente con il singolo sensore. La proponente ha affermato che per fare ciò servirebbe andare a realizzare un flusso inverso (da sistema di controllo a sensore) e dunque non si tratta di un requisito necessario ma al più opzionale. Tralatro viene affermato che solitamente questi strumenti, denominati _Asset Management_ non vengono realizzati all'interno di strumenti di _dashboarding_, ma in sezioni separate.
+- *Sistemi di Allerting*: la proponente ha invitato il team a riflettere sulle modalità di Allerting in quanto non erano state definite all'interno del documento. Si sono poi discusse alcune strategie per poter generare dati che effettivamente potessero andare ad innescare questi sistemi. La proponente afferma che tali dati non siano necessariamente da ricondurre alla produzione effettuata dai simulatore, ma possano che essere trasmessi a Kafka manualmente. 
+
+Non è stata espressa una vera e propria opinione, ma i requisiti riportati nel documento sembrano essere in linea con l'idea della proponente. 
+
+=== Workshop su Docker
+
+Alla luce del fatto che la quasi totalità dei componenti del nostro team ritiene che le proprie competenze in ambito Docker e tecnologie di containerizzazione siano ridotte, e che l'impiego di tali tecnologie rimane fondamentale al fine della buona riuscita del progetto, è stata avanzata l'idea di organizzare un workshop di allineamento sul tema in maniera tale da colmare le lacune esistenti. 
+La proponente ha accettato la proposta e ne è stata fissata la data: il 29 novembre alle ore 15.30. 
+
+
+=== Obbiettivi dello Sprint
+L'intenzione iniziale della proponente per questo sprint era quello di introdurre nel sistema la componente di persistenza dei dati simulati.
+
+Confermata l'intenzione del team di effettuare la prima revisione entro il 18 dicembre, è stato invece deciso di accorpare nel backlog dello sprint anche la componente di visualizzazione che era stata inizialmente pianificata come parte dello sprint successivo. Presumibilmente entro il 7 dicembre il team avrà quindì realizzato una prima versione completa del _Proof of Concept_. 
```

Comment:
- "Una volta confermata l'intenzione del team.."
- "accorpare nello _Sprint backlog_"
- toglierei il "Presumibilmente": "In altre parole, il team avrà realizzato una prima versione completa del _Proof of Concept_ entro la fine dello sprint attuale, programmata per il 7 Dicembre."